src/main/java/com/sleekbyte/tailor/listeners/whitespace/ParenthesisWhitespaceListener.java

```diff
@@ -0,0 +1,49 @@
+package com.sleekbyte.tailor.listeners.whitespace;
+
+import com.sleekbyte.tailor.antlr.SwiftBaseListener;
+import com.sleekbyte.tailor.antlr.SwiftParser;
+import com.sleekbyte.tailor.common.Rules;
+import com.sleekbyte.tailor.output.Printer;
+import com.sleekbyte.tailor.utils.WhitespaceVerifier;
+
+/**
+ * Parse tree listener to flag illegal whitespace in parentheses.
+ */
+public final class ParenthesisWhitespaceListener extends SwiftBaseListener {
+    private WhitespaceVerifier verifier;
```

Comment:
could you move both the helper methods into this class, I don't think they're needed anywhere else?